tests/CMakeLists.txt

```diff
@@ -170,21 +167,19 @@ add_legacy_tests(
   igraph_knn
   igraph_local_transitivity
   igraph_minimum_spanning_tree
+  igraph_pagerank
+  igraph_radius
   igraph_reciprocity
   igraph_similarity
   igraph_simplify
   igraph_transitive_closure_dag
   igraph_transitivity
+  single_target_shortest_path
 )
 if (BUILD_INTERNAL_TESTS)
   add_legacy_tests(
     FOLDER simple NAMES
-    igraph_eccentricity
-    igraph_get_all_simple_paths
-    igraph_pagerank
-    igraph_radius
-    igraph_rewire
-    single_target_shortest_path
+    igraph_rewire # Uses internal igraph_rewire_core
```

Comment:
One small note is to eventually use the correct name in the comment, since @ntamas already renamed `igraph_rewire_core`